docs/cases/cases-overview.asciidoc

```diff
@@ -102,3 +102,45 @@ NOTE: Once a visualization has been added to a case, it cannot be modified or de
   
 [role="screenshot"]
 image::images/cases-open-vis.png[Shows where the Open Visualization option is]
+
+[float]
+[[cases-export-import]]
+== Export and import cases
+
+Cases can be <<cases-export, exported>> and <<cases-import, imported>> as saved objects through the Kibana {kibana-ref}/managing-saved-objects.html[Saved Objects] UI.
+
+IMPORTANT: Before importing a Lens visualization into a space, ensure the visualization's data is present. Without it, the Lens visualization won't work.
+
+[float]
+[[cases-export]]
+=== Export a case
+Exporting cases can help you back up your cases or move them from one space to another. When you export a case, the case details, user actions, and comments are exported to a delimited JSON (ndjson) file. During the export process, text string comments, case alerts, and lens visualizations are exported.
```

Comment:
I think instead of backup (my mistake 🙂 for that) it is better to say `you can move objects between different Kibana instances` as stated here: https://security-docs_1199.docs-preview.app.elstc.co/guide/en/kibana/master/managing-saved-objects.html#managing-saved-objects-export-objects